lib/node_modules/@stdlib/math/base/special/cinvf/README.md

```diff
@@ -0,0 +1,251 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2025 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# cinvf
+
+> Compute the inverse of a single-precision complex floating-point number.
+
+<section class="intro">
+
+The inverse (or reciprocal) of a non-zero complex number `z = a + bi` is defined as
+
+<!-- <equation class="equation" label="eq:complex_inverse" align="center" raw="{\frac {1}{z}}=\frac{\bar{z}}{z{\bar{z}}} = \frac{a}{a^{2}+b^{2}} - \frac{b}{a^2+b^2}i." alt="Complex Inverse" > -->
+
+```math
+{\frac {1}{z}}=\frac{\bar{z}}{z{\bar{z}}} = \frac{a}{a^{2}+b^{2}} - \frac{b}{a^2+b^2}i.
+```
+
+<!-- </equation> -->
+
+</section>
+
+<!-- /.intro -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var cinvf = require( '@stdlib/math/base/special/cinvf' );
+```
+
+#### cinvf( z )
+
+Computes the inverse of a single-precision complex floating-point number.
+
+```javascript
+var Complex64 = require( '@stdlib/complex/float32/ctor' );
+var realf = require( '@stdlib/complex/float32/real' );
+var imagf = require( '@stdlib/complex/float32/imag' );
+
+var v = cinvf( new Complex64( 2.0, 4.0 ) );
+// returns <Complex64>
+
+var re = realf( v );
+// returns ~0.1
+
+var im = imagf( v );
+// returns ~-0.2
+```
+
+</section>
+
+<!-- /.usage -->
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var Complex64 = require( '@stdlib/complex/float32/ctor' );
+var uniform = require( '@stdlib/random/base/uniform' );
+var cinvf = require( '@stdlib/math/base/special/cinvf' );
+
+var z1;
```

Comment:
@anandkaranubc Would you mind refactoring to use `filledarrayBy` and `logEachMap` here? This way we can avoid the manual for loop.